beacon-chain/rpc/eth/light-client/handlers.go

```diff
@@ -46,18 +47,62 @@ func (s *Server) GetLightClientBootstrap(w http.ResponseWriter, req *http.Reques
 		return
 	}
 
-	bootstrap, err := createLightClientBootstrap(ctx, state)
-	if err != nil {
-		httputil.HandleError(w, "could not get light client bootstrap: "+err.Error(), http.StatusInternalServerError)
+	switch blk.Version() {
+	case version.Phase0:
+		httputil.HandleError(w, "light client bootstrap is not supported for phase0", http.StatusNotImplemented)
 		return
-	}
+	case version.Altair:
+		bootstrap, err := createLightClientBootstrap(ctx, state)
+		if err != nil {
+			httputil.HandleError(w, "could not get light client bootstrap: "+err.Error(), http.StatusInternalServerError)
+			return
+		}
 
-	response := &structs.LightClientBootstrapResponse{
-		Version: version.String(blk.Version()),
-		Data:    bootstrap,
-	}
+		response := &structs.LightClientBootstrapResponse{
+			Version: version.String(blk.Version()),
+			Data:    bootstrap,
+		}
+		w.Header().Set(api.VersionHeader, version.String(version.Altair))
 
-	httputil.WriteJson(w, response)
+		httputil.WriteJson(w, response)
+		return
+	case version.Bellatrix:
+		httputil.HandleError(w, "light client bootstrap is not supported for bellatrix", http.StatusNotImplemented)
+		return
+	case version.Capella:
+		bootstrap, err := createLightClientBootstrapCapella(ctx, state)
+		if err != nil {
+			httputil.HandleError(w, "could not get light client bootstrap: "+err.Error(), http.StatusInternalServerError)
+			return
+		}
+
+		response := &structs.LightClientBootstrapResponseCapella{
+			Version: version.String(blk.Version()),
+			Data:    bootstrap,
+		}
+		w.Header().Set(api.VersionHeader, version.String(version.Capella))
+
+		httputil.WriteJson(w, response)
+		return
+	case version.Deneb:
+		bootstrap, err := createLightClientBootstrapDeneb(ctx, state)
+		if err != nil {
+			httputil.HandleError(w, "could not get light client bootstrap: "+err.Error(), http.StatusInternalServerError)
+			return
+		}
+
+		response := &structs.LightClientBootstrapResponseDeneb{
+			Version: version.String(blk.Version()),
+			Data:    bootstrap,
+		}
+		w.Header().Set(api.VersionHeader, version.String(version.Deneb))
+
+		httputil.WriteJson(w, response)
+		return
+	case version.Electra:
+		httputil.HandleError(w, "light client bootstrap is not supported for electra", http.StatusNotImplemented)
```

Comment:
The HTTP code should be `http.BadRequest`. `StatusNotImplemented` means that the server doesn't have some functionality implemented, but there is no functionality to implement here.

Reply:
either is fine, I guess